tests/integration_tests/test_ingest_component_pathfinders.py

```diff
@@ -134,6 +134,16 @@ def test_ingest_pf_r1_file_success_with_tf_data_already_in(
     # check submission id correctly generated
     assert len(Submission.query.filter(Submission.submission_id == "S-PF-R01-1").all()) == 1
 
+    # TODO FMD-260: remove after this is enforced via DB constraint and tested elsewhere
+    assert (
+        ProgrammeJunction.query.join(Submission, Submission.id == ProgrammeJunction.submission_id)
+        .join(Programme, Programme.id == ProgrammeJunction.programme_id)
+        .filter(Submission.submission_id == "S-PF-R01-1", Programme.programme_id == "PF-BOL")
+        .first()
```

Comment:
should we use one() here instead? Non a massive deal, but we've had a few instances now where we use first() when really we only expect one row, but then due to some later misuse of test fixtures elsewhere, we get test leakage and more than one row returning, meaning first() is no longer necessarily getting the row you want, and when it doesn't the error message can be confusing. I think if we expect one row, always best to use one()

Reply:
Sure - although this should only be around for one or two PRs